specification/azurestack/resource-manager/Microsoft.AzureStack/stable/2017-06-01/CloudManifestFile.json

```diff
@@ -0,0 +1,53 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2017-06-01",
+    "title": "Azure Stack Azure Bridge Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/providers/Microsoft.AzureStack/cloudmanifestfile": {
```

Comment:
This does not look like complete path. should start with `/subscription/...`

Reply:
This is a tenant level operation.

Reply:
I see. Can you also add example file reference? Thanks.

Reply:
Example added